Production Merge: Latest code from staging to production

supriyartambewaiin opened the pull request:
No Screenshot

alok-nakhate reviewed (approved):
Changes are reviewed and approved.

nknikhilkasar committed:
Merge pull request #1373 from Wai-Technologies/users/nikhil/bug-1368-password-and-confirm-password-validation

Resolved Bug 1368: Design System > Components > Password Setting > Default > The new password and confirm new password validation are not displayed properly.

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/bug-1359-light-theme-selected-text-is-not-display

nknikhilkasar committed:
Merge pull request #1377 from Wai-Technologies/users/nikhil/bug-1359-light-theme-selected-text-is-not-display

Bug-1359: Fixed selected text not displayed issue.

rohit9096 committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

Kushal-Firodiya committed:
Merge pull request #1380 from Wai-Technologies/users/deepashri/task-move-dev-changes-to-stage-branch-09-01-25

Merging Development : Moving Changes from Development to Stage

VenuKG committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

VenuKG committed:
Merge branch 'development' into users/venu/bug-1264-deign-system-sidebar-element-chart-map-icon-issue

VenuKG committed:
Merge branch 'development' into users/venu/bug-1251-design-system-component-chatbot-overlapping-issue-and-popup-closing-issue

VenuKG committed:
Merge branch 'development' into users/venu/bug-1213-design-system-element-navtabs-icon-icon-only-styled-vertical-left-align-alignment-issue

VenuKG committed:
Merge branch 'development' into users/venu/bug-1281-design-system-element-input-number-input-overlapping-issue

atharvamachave28 committed:
Merge branch 'development' into users/atharva/708-task-check-token-and-replace-where-needed

atharvamachave28 committed:
Merge pull request #1307 from Wai-Technologies/users/atharva/708-task-check-token-and-replace-where-needed

Resolved Bug 823 : Design System > Create color variants for toast elements

VenuKG committed:
commit changes for remove unwantes code in rds-input.txs for password

atharvamachave28 committed:
Merge branch 'development' into users/atharva/bug-1260/element-badge-color-variant-issue

VenuKG committed:
Merge branch 'development' into users/venu/bug-1213-design-system-element-navtabs-icon-icon-only-styled-vertical-left-align-alignment-issue

nknikhilkasar committed:
Merge branch 'development' into users/venu/bug-1264-deign-system-sidebar-element-chart-map-icon-issue

VenuKG committed:
Merge pull request #1280 from Wai-Technologies/users/venu/bug-1213-design-system-element-navtabs-icon-icon-only-styled-vertical-left-align-alignment-issue

Resolved Bug 1213 : Design System > Elements > Navtabs > Icon Only > Styled > Vertical left filled > Not able to see home icon after hovering on it

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/bug-1375-sub-element-name-change

nknikhilkasar committed:
Merge pull request #1379 from Wai-Technologies/users/nikhil/bug-1375-sub-element-name-change

Resolved Bug 1375: Design System > Elements/Components >The sub-element/component name is the same as the main component/element

VenuKG committed:
commit changes for alignment for fileupload and emoji picker icon

nknikhilkasar committed:
Merge branch 'development' into users/venu/bug-1251-design-system-component-chatbot-overlapping-issue-and-popup-closing-issue

nknikhilkasar committed:
Merge branch 'development' into users/venu/bug-1281-design-system-element-input-number-input-overlapping-issue

SoheshMG committed:
Removed hardcoded values and added variables for colours

atharvamachave28 committed:
Merge branch 'development' into users/atharva/bug-1260/element-badge-color-variant-issue

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

atharvamachave28 committed:
Merge pull request #1342 from Wai-Technologies/users/atharva/bug-1260/element-badge-color-variant-issue

Resolved Bug 1260 : Design System > Elements > Badge > In Color variant info color is not according to Figma design.

VenuKG committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

nknikhilkasar committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

VenuKG committed:
Merge pull request #1284 from Wai-Technologies/users/venu/bug-1281-design-system-element-input-number-input-overlapping-issue

Resolved Bug 1281:Design System > Element > Input > Number Input > Overlapping issue

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

VenuKG committed:
Merge pull request #1262 from Wai-Technologies/users/venu/bug-1251-design-system-component-chatbot-overlapping-issue-and-popup-closing-issue

Resolved Bug 1251 : Design System > Component > Chatbot > Overlapping issue and pop-up closing issue

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

VenuKG committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

SoheshMG committed:
Merge pull request #1381 from Wai-Technologies/users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

Resolved Bug 1344 : Design System > Components > Claims And Developer Mode > Default > Input box highlighted in required color

VenuKG committed:
Merge pull request #1078 from Wai-Technologies/users/deepashri/task-776-enhancements-in-toaster

Completed Task 776 : Design System > Elements > Enhancements in Toast

VenuKG committed:
Merge pull request #1295 from Wai-Technologies/users/venu/bug-1264-deign-system-sidebar-element-chart-map-icon-issue

Resolved Bug 1264 : Design System > Sidebar > Elements > Chart & Map Icon issue

SoheshMG committed:
Merge branch 'development' into users/sohesh/enhancement-975-element-counter-enhancement-in-counter

nknikhilkasar committed:
Enhancement-1174: Added a functionality for left, right and no pin for grid table column.

SoheshMG committed:
Enhancement-975: Remove conflicts for colorvariables-dark.scss

SoheshMG committed:
Added variables and removed hardcoded colors and removed commented code

Sachin-Ghorpade committed:
Merge pull request #1306 from Wai-Technologies/users/sachin/bug-1274-components-admin-dashboard-ui-alignment-issues

Resolved Bug 1274: Components Admin Dashboard UI & alignment Issues.

Sachin-Ghorpade committed:
Merge pull request #1301 from Wai-Technologies/users/sachin/bug-1282-elements-checkbox-check-box-texts-spacing-issue

Resolved Bug 1282: Elements Checkbox Check box & texts spacing issue

Sachin-Ghorpade committed:
Merge branch 'development' into users/sachin/bug-1309-design-system-elements-histogram-canvas-does-not-display-proper

nknikhilkasar committed:
Merge branch 'development' into users/sohesh/enhancement-975-element-counter-enhancement-in-counter

SoheshMG committed:
Merge pull request #1382 from Wai-Technologies/users/sohesh/enhancement-975-element-counter-enhancement-in-counter

Resolved Bug 975 : Design System > Element > Counter > Enhanced Counter As Per Design

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/enhancement-1174-add-left-right-pin-functionality-for-grid

nknikhilkasar committed:
Enhancement-1174: update icon size for table header icons

nknikhilkasar committed:
Enhancement-1174: Fixed action header position

Sachin-Ghorpade committed:
Resolved Conflict

Sachin-Ghorpade committed:
Merge branch 'development' into users/sachin/bug-1309-design-system-elements-histogram-canvas-does-not-display-proper

Sachin-Ghorpade committed:
Merge pull request #1317 from Wai-Technologies/users/sachin/bug-1309-design-system-elements-histogram-canvas-does-not-display-proper

Resolved Bug 1309: Design System > Elements > Big Number > Histogram > Canvas does not display proper

Sachin-Ghorpade committed:
Merge branch 'development' into users/sachin/bug-1340-design-system-components-blog-post-label-missing-spacing-issue

Sachin-Ghorpade committed:
Merge pull request #1365 from Wai-Technologies/users/sachin/bug-1340-design-system-components-blog-post-label-missing-spacing-issue

Resolved Bug 1340: Design System > Components > Blog Post > Default > label is missing & Blog Id input box spacing issue

Sachin-Ghorpade committed:
Merge branch 'development' into users/sachin/bug-1349-design-system-components-properties-button-alignment-issue

Sachin-Ghorpade committed:
Merge pull request #1364 from Wai-Technologies/users/sachin/bug-1349-design-system-components-properties-button-alignment-issue

Resolved Bug 1349: Design System > Components > Properties > Both stories Add button alignment issue

Sachin-Ghorpade committed:
Merge pull request #1363 from Wai-Technologies/users/sachin/bug-1346-design-components-dynamic-entity-properties-label-input-box-alignment-issue

Resolved Bug 1346: Design System > Components > Dynamic Entity Properties > Default > Label & input box alignment issue

Sachin-Ghorpade committed:
Merge branch 'development' into users/sachin/bug-1148-design-system-webhook-subscription-required-text-alignment-issue

poojagurav21 committed:
Merge branch 'development' into users/nikhil/enhancement-1174-add-left-right-pin-functionality-for-grid

Sachin-Ghorpade committed:
Merge pull request #1299 from Wai-Technologies/users/sachin/bug-1148-design-system-webhook-subscription-required-text-alignment-issue

Resolved Bug 1148: Components Webhook Subscription Default Text alignment issue

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/enhancement-1174-add-left-right-pin-functionality-for-grid

SoheshMG committed:
updated the scss file for dropdown

SoheshMG committed:
Changes in .dropdown-menu.show

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1370-component-login-default-language-dropdown-issue-position-on-scrolling

SoheshMG committed:
Bug-1370: revert  back older code.

SoheshMG committed:
Merge pull request #1386 from Wai-Technologies/users/sohesh/bug-1370-component-login-default-language-dropdown-issue-position-on-scrolling

Resolved Bug 1370 : Design System > Components > Login > Default > Fixed Language Dropdown

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/enhancement-1174-add-left-right-pin-functionality-for-grid

poojagurav21 committed:
Merge pull request #1385 from Wai-Technologies/users/nikhil/enhancement-1174-add-left-right-pin-functionality-for-grid

Enhanced Task 1174 : Design System > Storybook > Add enhanced functionalities to Grid Component

sharmaamanwai committed:
Update storybookreact.yml

sharmaamanwai committed:
Merge pull request #1388 from Wai-Technologies/sharmaamanwai-patch-1

Update storybookreact.yml

sharmaamanwai committed:
Update storybookreact.yml

sharmaamanwai committed:
Merge pull request #1391 from Wai-Technologies/sharmaamanwai-patch-2

Update storybookreact.yml

SoheshMG committed:
Added padding and margin to custom-theme.css

SoheshMG committed:
product 1 changes for mobile view

VenuKG committed:
commit changes for update in DataTable- add isClickable property

VenuKG committed:
commit changes for remove comment

nitinsonawane123 committed:
Commit Changes for task 1326

SoheshMG committed:
Update css file to and 40px height added

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1241-components-add-validations

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1361-elements-file-uploader-is-not-display-proper

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-components-tenant-information-after-submitting-field-not-clear

nitinsonawane123 committed:
Merge pull request #1366 from Wai-Technologies/users/nitin/bug-1353-components-tenant-information-after-submitting-field-not-clear

Resolved bug 1353 : Design System > Components > Tenant Information > Default > After submitting field not clear.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1106-component-address-input-box-alignment-issue

nitinsonawane123 committed:
Merge pull request #1357 from Wai-Technologies/users/nitin/bug-1106-component-address-input-box-alignment-issue

Resolved bug 1106 : Design System > Components > Address Input > Default > Input box alignment issue.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1327-components-dialog-cross-button-alignment-issue

nitinsonawane123 committed:
Merge pull request #1329 from Wai-Technologies/users/nitin/bug-1327-components-dialog-cross-button-alignment-issue

Resolved bug 1327 : Design System > Components >Dialog > Default > Cross button alignment issue.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1360-elements-fab-menu-small-mobile-view-is-not-display-proper

nitinsonawane123 committed:
Merge pull request #1362 from Wai-Technologies/users/nitin/bug-1360-elements-fab-menu-small-mobile-view-is-not-display-proper

Resolved bug 1360 : Design System > Elements > Fab Menu > Small mobile view is not display proper.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1348-components-personal-info-verify-email-button-alignment-issue

nitinsonawane123 committed:
Merge pull request #1358 from Wai-Technologies/users/nitin/bug-1348-components-personal-info-verify-email-button-alignment-issue

Resolved bug 1348 : Design System > Components > Personal Info > Default > Verify Email button alignment issue.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1207-component-blog-post-not-able-to-see-last-field-of-form-and-button-size-is-bigger

nitinsonawane123 committed:
Merge pull request #1279 from Wai-Technologies/users/nitin/bug-1207-component-blog-post-not-able-to-see-last-field-of-form-and-button-size-is-bigger

Resolved bug 1207 : Design System > Component > Blog Post > Default > Not able to see last field of form and button size is bigger than field.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1107-chat-placeholder-text-icon-not-centralized

nitinsonawane123 committed:
Merge pull request #1355 from Wai-Technologies/users/nitin/bug-1107-chat-placeholder-text-icon-not-centralized

Resolved bug 1107 : Design System > Components > Chat > Default > Placeholder text & icon is not centralized.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1124-change-size-preview-large-and-tablet-view-top-space-is-not-display

nitinsonawane123 committed:
Merge pull request #1336 from Wai-Technologies/users/nitin/bug-1124-change-size-preview-large-and-tablet-view-top-space-is-not-display

Resolved bug 1124 : Design System > Component > Default > Change the size preview > Large & Tablet view top space is not display.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1126-components-edition-dark-theme-basics-and-features-text-color-issue

nitinsonawane123 committed:
Merge pull request #1334 from Wai-Technologies/users/nitin/bug-1126-components-edition-dark-theme-basics-and-features-text-color-issue

Resolved bug 1126 : Design System > Components > Edition > Default > Dark theme Basics & Features text color issue.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1322-components-email-settings-label-input-box-alignment-issue

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1129-components-identity-client-basic-required-consent-alignment-issue

poojagurav21 committed:
Merge pull request #1314 from Wai-Technologies/users/nitin/bug-components-identity-client-basic-required-consent-alignment-issue

Resolved bug 1129 : Design System > Components > Identity Client Basic > Default > Required Consent alignment issue.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1149-application-shells-ui-issue

nitinsonawane123 committed:
Commit Changes for bug 1322

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1137-component-register-toggle-is-display-outside-the-modal

nitinsonawane123 committed:
Merge pull request #1275 from Wai-Technologies/users/nitin/bug-1137-component-register-toggle-is-display-outside-the-modal

Resolved bug 1137 : Design System > Components > Register > Default >Toggle is display outside of modal.

kravi97 committed:
Completed Bug : Fixed the position of Rating stars

kravi97 committed:
Merge branch 'development' into user/ravindra/fixed-rating-star-issue

kravi97 committed:
changes for reviews category rating alignment.

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1384-design-system-logo-border-not-displayed-properly

kravi97 committed:
Completed bug 1395.

SoheshMG committed:
Revert "product 1 changes for mobile view"

This reverts commit fb4d1df95248d80d7e065fef59d60a513310810b.

nitinsonawane123 committed:
Commit Changes for bug 1354

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1149-application-shells-ui-issue

nitinsonawane123 committed:
Merge pull request #1320 from Wai-Technologies/users/nitin/bug-1149-application-shells-ui-issue

Resolved bug 1149 : Design System > Application Shells > Default > UI issue.

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/task-1232-application-enhancement-in-comment-file-uploader-component-accordion-position-and-nav-tab

nitinsonawane123 committed:
Merge pull request #1326 from Wai-Technologies/users/nitin/task-1232-application-enhancement-in-comment-file-uploader-component-accordion-position-and-nav-tab

Completed task 1232 : Design System > Application > Enhancement in Comment component, File Uploader component, Accordion position and Nav tab badges

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1241-components-add-validations

nitinsonawane123 committed:
Commit Changes for margin issue

SoheshMG committed:
Merge pull request #1396 from Wai-Technologies/users/sohesh/bug-1384-design-system-logo-border-not-displayed-properly

Resolved Bug 1384 : Design System > Logo Border Not Displayed Properly

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1389-design-system-elements-review-category-center-aligned-left-aligned-stories-profile-icon

nitinsonawane123 committed:
Commit Changes for bug 1347

poojagurav21 committed:
Merge branch 'development' into users/venu/bug-1390-design-system-datatable-enhancement-add-property-isclickable

poojagurav21 committed:
changes for build issue

poojagurav21 committed:
Merge pull request #1392 from Wai-Technologies/users/venu/bug-1390-design-system-datatable-enhancement-add-property-isclickable

Resolved Bug 1390 : Design System > Component > Data Table > Enhancement > Add property Is clickable to column [New Project Flow]

SoheshMG committed:
Revert "Added regex comment if only alphanumeric required with 8 characters long"

This reverts commit 6fac7c04c1b44445c0713433856132c56022f354.

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

SoheshMG committed:
Merge branch 'users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements' of https://github.com/Wai-Technologies/raaghu-react into users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

SoheshMG committed:
Changed rds-comp-register.tsx file to changed padding

SoheshMG committed:
Merge pull request #1352 from Wai-Technologies/users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

Resolved Bug 1242 : Design system > Elements > Feed > Added size parameter and Resolved other Bugs

poojagurav21 committed:
Merge branch 'development' into users/sohesh/bug-1389-design-system-elements-review-category-center-aligned-left-aligned-stories-profile-icon

kravi97 committed:
Merge pull request #1394 from Wai-Technologies/user/ravindra/fixed-rating-star-issue

Completed Bug : Design system > Elements > Rating > The position of the rating stars is inconsistent after changing the count #1393

kravi97 committed:
Merge pull request #1397 from Wai-Technologies/user/ravindra/time-picker-icon-functionality

Completed bug 1395 : Design System > Elements >Time Picker > Default > Outside icon click functionality is not working.

poojagurav21 committed:
Merge branch 'development' into user/pooja/task-dev-to-stage

poojagurav21 committed:
Merge branch 'development' into users/sohesh/bug-1389-design-system-elements-review-category-center-aligned-left-aligned-stories-profile-icon

poojagurav21 committed:
Merge pull request #1399 from Wai-Technologies/users/sohesh/bug-1389-design-system-elements-review-category-center-aligned-left-aligned-stories-profile-icon

Resolved Bug 1389 : Design System > Elements > Review Category > Center Aligned/Left Aligned Stories Profile Icon Is Enlarged

poojagurav21 committed:
Merge branch 'development' into users/nitin/bug-1241-components-add-validations

poojagurav21 committed:
Merge pull request #1403 from Wai-Technologies/user/pooja/task-dev-to-stage

Task Merge development to stage

Shankar1006 committed:
Update storybookreact.yml

Shankar1006 committed:
Update storybookreact.yml

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1241-components-add-validations

SoheshMG committed:
updated the value array and changed position of  previous buttons

nitinsonawane123 committed:
Commit Changes for bug 1353

SoheshMG committed:
updated the maxRecordsPage condition

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/task-849-element-progress-bar-enhancements-in-style

nitinsonawane123 committed:
Commit Changes for bug 1348

nitinsonawane123 committed:
Commit Changes for bug 1360

nitinsonawane123 committed:
Merge pull request #1323 from Wai-Technologies/users/nitin/bug-1241-components-add-validations

Resolved bug 1241 : Design System > Components > Add validations.

VenuKG committed:
Merge branch 'development' into users/venu/design-system-deployement-dev-to-stage

VenuKG committed:
Merge pull request #1411 from Wai-Technologies/users/venu/design-system-deployement-dev-to-stage

Design System Deployment development to staging

nknikhilkasar committed:
Add advance stepper functionality with customizable properties

SoheshMG committed:
Reopen Feed changes added size property

SoheshMG committed:
Reopen adjusted buttons in Premium Support

SoheshMG committed:
Reopen personal info button alignment

nitinsonawane123 committed:
Commit Changes for task 784

Dipak7028 committed:
updated rds comp grid tsx file

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1404-design-system-components-grid-overlapping-issue

nknikhilkasar committed:
Merge pull request #1413 from Wai-Technologies/user/nikhil/enhance-1387-add-advance-stepper

Enhanced 1387: Design System > Elements > Stepper > Enhancement in stepper

nitinsonawane123 committed:
Commit Changes for bug 1412

Sachin-Ghorpade committed:
Committed changes on banner light theme text is not display

nknikhilkasar committed:
Enhance password validation logic and add confirmation checks in password settings component

nknikhilkasar committed:
Merge branch 'development' into users/dipak/bug-1404-design-system-components-grid-overlapping-issue

Dipak7028 committed:
Merge pull request #1415 from Wai-Technologies/users/dipak/bug-1404-design-system-components-grid-overlapping-issue

Resolved Bug 1404 :  Design System > Components > Grid > Overlapping issue

nitinsonawane123 committed:
Merge branch 'development' into users/sachin/bug-1282-design-system-baner-light-theme-text-is-not-display

Sachin-Ghorpade committed:
Merge pull request #1419 from Wai-Technologies/users/sachin/bug-1282-design-system-baner-light-theme-text-is-not-display

Resolved Bug 1282: Design System > Baner > Raaghu > light theme text is not display

nitinsonawane123 committed:
Merge branch 'development' into users/nikhil/bug-1368-password-validation-for-current-password-and-new-password

nknikhilkasar committed:
bug 1368 changed variable name

Sachin-Ghorpade committed:
Committed changes on  Cover Image  Blog Id input box spacing issue

nknikhilkasar committed:
Merge pull request #1420 from Wai-Technologies/users/nikhil/bug-1368-password-validation-for-current-password-and-new-password

Resolved Bug 1368 : Design System > Components > Password Setting > Default > The new password and confirm new password validation are not displayed properly.

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1348-component-personal-info-verify-email-button-alignment-issue

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1311-design-system-elements-pagination-advanced-dark-theme-pagination-is-not-display-proper

nknikhilkasar committed:
Merge branch 'development' into users/nitin/task-849-element-progress-bar-enhancements-in-style

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1360-elemets-fab-menu-small-mobile-view-is-not-display-proper

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1412-element-dropdown-multiselect-selected-option-is-not-display-proper

SoheshMG committed:
Merge pull request #1423 from Wai-Technologies/users/sohesh/bug-1311-design-system-elements-pagination-advanced-dark-theme-pagination-is-not-display-proper

Resolved Bug 1311 : Design System > Elements > Pagination > Advanced > Dark theme pagination is now display proper

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1347-components-page-not-found-text-is-not-displaying-center

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1348-component-personal-info-verify-email-button-alignment-issue

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1360-elemets-fab-menu-small-mobile-view-is-not-display-proper

nitinsonawane123 committed:
Commit Changes for bug 1348

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1361-elements-file-uploader-is-not-display-proper

nitinsonawane123 committed:
resolved conflict

nknikhilkasar committed:
Merge branch 'development' into user/nitin/task-784-elements-enhancement-in-action-button-dropdown

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

Dipak7028 committed:
Resolve issues in Elements >Notification > Default > Mark As Read button alignment issue.

Resolved issue by  Mark As Read button alignment issue in changing by m-3 to m-2

nknikhilkasar committed:
Merge branch 'development' into users/sachin/bug-1340-design-system-components-blog-post-default-cover-image-blog-id-input-box-spacing-issue

Dipak7028 committed:
Merge pull request #1425 from Wai-Technologies/users/dipak/1395-design-system-elements-time-picker-default-outside-icon-click-functionality-is-not-working

Resolved bug 1395: Elements >Notification > Default > Mark As Read button alignment issue.

nitinsonawane123 committed:
Merge pull request #1408 from Wai-Technologies/users/nitin/bug-1360-elemets-fab-menu-small-mobile-view-is-not-display-proper

Resolved bug 1360 : Design System > Elements > Fab Menu > Small mobile view is not display proper.

nitinsonawane123 committed:
Merge pull request #1406 from Wai-Technologies/users/nitin/bug-1348-component-personal-info-verify-email-button-alignment-issue

Resolved bug 1348 : Design System > Components > Personal Info > Default > Verify Email button alignment issue.

nitinsonawane123 committed:
Merge pull request #1400 from Wai-Technologies/users/nitin/bug-1347-components-page-not-found-text-is-not-displaying-center

Resolved bug 1347 : Design System > Components > Page not found > Default > Text is not displaying center.

nitinsonawane123 committed:
Merge pull request #1325 from Wai-Technologies/users/nitin/bug-1322-components-email-settings-label-input-box-alignment-issue

Resolved bug 1322 : Design System > Components > Email Settings > Default > Label & Input box alignment issue.

Sachin-Ghorpade committed:
Merge branch 'development' into users/sachin/bug-1340-design-system-components-blog-post-default-cover-image-blog-id-input-box-spacing-issue

Sachin-Ghorpade committed:
Merge pull request #1421 from Wai-Technologies/users/sachin/bug-1340-design-system-components-blog-post-default-cover-image-blog-id-input-box-spacing-issue

Resolved Bug 1340: Design System > Components > Blog Post > Default > Cover Image & Blog Id input box spacing issue.

nitinsonawane123 committed:
Commit Changes for bug 1353

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1412-element-dropdown-multiselect-selected-option-is-not-display-proper

Dipak7028 committed:
Resolved Bug 1404 : Design System > Elements > Rating > The position of the rating stars is inconsistent after changing the count

Resolve issues in  Elements > Rating > The position of the rating stars is inconsistent after changing the count

Resolved issue by add a justify-content-between and remove the pe-5 in class

nitinsonawane123 committed:
Commit Changes for bug 1412

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

rameshwarkamandar committed:
Add missing controls, update interface

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/task-849-element-progress-bar-enhancements-in-style

nitinsonawane123 committed:
Merge pull request #1300 from Wai-Technologies/users/nitin/task-849-element-progress-bar-enhancements-in-style

Completed Task 849 : Design System > Element > Progress Bar > Default : Enhancements in style.

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1393-design-system-Elements-Rating-the-position-of-the-rating-stars-is-inconsistent-after-changing-the-count

nknikhilkasar committed:
Merge branch 'development' into users/rameshwar/bug/1427-elements-app-detail-Missing-controls-and-missing-formatting

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-1412-element-dropdown-multiselect-selected-option-is-not-display-proper

Dipak7028 committed:
Merge pull request #1426 from Wai-Technologies/users/dipak/bug-1393-design-system-Elements-Rating-the-position-of-the-rating-stars-is-inconsistent-after-changing-the-count

Resolved Bug 1404 : Design System > Elements > Rating > The position of the rating stars is inconsistent after changing the count

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

nitinsonawane123 committed:
Merge pull request #1418 from Wai-Technologies/users/nitin/bug-1412-element-dropdown-multiselect-selected-option-is-not-display-proper

Resolved bug 1412 : Design System > Elements > Dropdown > With Multi Select > Small & large mobile selected option is not display proper.

SoheshMG committed:
changes in rds-select-list.css

nitinsonawane123 committed:
Commit Changes for bug 1354

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1361-elements-file-uploader-is-not-display-proper

nitinsonawane123 committed:
Merge pull request #1367 from Wai-Technologies/users/nitin/bug-1361-elements-file-uploader-is-not-display-proper

Resolved bug 1361 : Design System > Elements > File Uploader > Default > File Uploader is not display proper.

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

nitinsonawane123 committed:
Merge pull request #1398 from Wai-Technologies/users/nitin/bug-1354-components-visual-setting-after-submitting-select-list-values-are-not-clear

Resolved bug 1354 : Design System > Components > Visual Setting > Default > After submitting select list values are not clear.

rameshwarkamandar committed:
Added sample link url

nknikhilkasar committed:
Merge branch 'development' into users/rameshwar/bug/1427-elements-app-detail-Missing-controls-and-missing-formatting

rameshwarkamandar committed:
Merge pull request #1428 from Wai-Technologies/users/rameshwar/bug/1427-elements-app-detail-Missing-controls-and-missing-formatting

Resolved Bug 1427: Design System > Element > App detail > Default > Missing Controls and missing formatting.

nitinsonawane123 committed:
Commit Changes for bug 1353

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

Sachin-Ghorpade committed:
Committed changes on checkbox and text alignment issue and validation pattern

nitinsonawane123 committed:
Merge pull request #1405 from Wai-Technologies/users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

Resolved bug 1353 : Design System > Components > Tenant Information > Default > After submitting field not clear.

Sachin-Ghorpade committed:
Committed changes on Validation message should be same as per other components

Sachin-Ghorpade committed:
Committed changes on validation msg

supriyartambewaiin committed:
Merge branch 'development' into users/sachin/bug-1346-design-system-components-database-connection-checkbox-and-text-alignment-issue

Sachin-Ghorpade committed:
Merge pull request #1433 from Wai-Technologies/users/sachin/bug-1346-design-system-components-database-connection-checkbox-and-text-alignment-issue

Resolved Bug 1346: Design System > Components >Database Connection > checkbox and text alignment issue and Database URL validation

SoheshMG committed:
Changed RdsTextArea to RdsTextEditor

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

SoheshMG committed:
Merge pull request #1429 from Wai-Technologies/users/sohesh/bug-1242-add-parameter-size-for-avatar-and-updating-other-elements

Resolved Bug 1242 : Design system > Elements > Feed > Added size parameter and Resolved Reopen Bugs

supriyartambewaiin committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

rameshwarkamandar committed:
Resolved bug: 1434 -Code formatting and optimization.

Dipak7028 committed:
Resolved Bug 1293: Design system > Components > Edition Information > Small large tablet button alignment

1395 Resolve issues in design ystem components edition information small large tablet button alignment issue.

nitinsonawane123 committed:
Merge branch 'development' into user/nitin/task-784-elements-enhancement-in-action-button-dropdown

Dipak7028 committed:
Resolved Bug 1293: Design system > Components > Edition Information >Small large tablet button alignment

1395 Resolve issues in design ystem components edition information small l arge tablet button alignment issue.

-Resolved issue by Mark As small large tablet button alignment issue alignment issue in changed
Edition Information > Small, Large, Tablet button alignment should be proper.
Email Setting >Default > All view alignment should be proper.

Screenshot

Dipak7028 committed:
Merge pull request #1436 from Wai-Technologies/users/dipak/1293-design-system-components-edition-information-small-large-tablet-button-alignment-issue

Resolved Bug 1293: Design system > Components > Edition Information >Small large tablet button alignment

SoheshMG committed:
Merge branch 'development' into users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

SoheshMG committed:
Merge pull request #1431 from Wai-Technologies/users/sohesh/bug-1344-claims-default-input-box-highlighted-in-different-color

Resolved Bug 1344 : Design System > Components > Claims And Developer Mode > Default > Input box highlighted in required color Reopen Bug

rameshwarkamandar committed:
Merge branch 'development' into user/rameshwar/bug/1434-elements-alerts-Code-formatting-and-optimization

rameshwarkamandar committed:
Merge pull request #1435 from Wai-Technologies/user/rameshwar/bug/1434-elements-alerts-Code-formatting-and-optimization

Bug 1434: Resolved Design System > Elements > Alerts > Code formatting and optimization

nitinsonawane123 committed:
Commit Changes for task 784

supriyartambewaiin committed:
Merge branch 'development' into user/nitin/task-784-elements-enhancement-in-action-button-dropdown

nitinsonawane123 committed:
Merge pull request #1414 from Wai-Technologies/user/nitin/task-784-elements-enhancement-in-action-button-dropdown

Completed Task 784 : Design System > Elements > Enhancements in Action Button/Dropdown.

rameshwarkamandar committed:
Fixed deployment issue

rameshwarkamandar committed:
Merge pull request #1437 from Wai-Technologies/users/rameshwar/fixed-deployment-issue

Fixed deployment issue

Dipak7028 committed:
Resolved Bug 1393: Design System > Elements > Rating > The position of the rating stars is inconsistent after changing the count

Dipak7028 committed:
Merge pull request #1438 from Wai-Technologies/users/dipak/bug-1393-design-system-Elements-Rating-the-position-of-the-rating-stars-is-inconsistent-after-changing

Resolved Bug 1393: Design System > Elements > Rating > The position of the rating stars is inconsistent after changing the count

VenuKG committed:
Merge branch 'development' into users/venu/design-system-deployment-dev-to-stage

VenuKG committed:
Merge pull request #1439 from Wai-Technologies/users/venu/design-system-deployment-dev-to-stage

Design System Deployment development to staging

supriyartambewaiin committed:
Merge branch 'production' into users/supriya/merge-production-into-staging-for-deployment
